packages/terra-search-field/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Moved `additionalInputAttributes` to the top of the attributes list.
```

Comment:
`additionalInputAttributes` is an internal instance variable. This entry may not make sense to consumers reading the changelog. Something along the lines of `Moved custom input attributes to be spread first on the input element` may make sense to users.

Reply:
Changed c92ed69